src/webgpu/shader/validation/parse/enable.spec.ts

```diff
@@ -95,11 +89,7 @@ g.test('enable')
   .beforeAllSubcases(t => {
     const features: GPUFeatureName[] = [];
     const name: string = t.params.case as string;
-    if (name.includes('subgroups_f16')) {
-      features.push('subgroups' as GPUFeatureName);
-      features.push('subgroups-f16' as GPUFeatureName);
-      features.push('shader-f16');
-    } else if (name.includes('subgroups')) {
+    if (name.includes('subgroups')) {
```

Comment:
This logic seems like it will fail the `subgroups_f16_pass1` and `subgroups_f16_pass2` tests because only the subgroups feature will be enabled.

Reply:
Yes fixed.